packages/commerce-sdk-react/src/auth/index.ts

```diff
@@ -361,6 +361,13 @@ class Auth {
         store.delete(key)
     }
 
+    private convertSecondsToDate(seconds: number | undefined): Date | undefined {
+        if (typeof seconds === 'number') {
+            return new Date(Date.now() + seconds * 1000)
+        }
+        return undefined
```

Comment:
What are your thoughts on throwing if the parameter isn't a number instead of returning a different type object?

Reply:
The`undefined` value is valid for `expires` cookie attribute in `js-cookie`. Creates a session cookie that expires when the browser is closed if the expires is undefined.
https://github.com/js-cookie/js-cookie/tree/latest?tab=readme-ov-file#expires

Reply:
I see.. I figured that was the case. Maybe I'm just being a little nit picky but the name of your function implies that you are going to return a date hence the comment. I suppose you can return undefined, just make sure you describe this functions behaviour in a js doc comment so others will understand how it works.